src/shivani/Assignment16/CharLength.java

```diff
@@ -0,0 +1,39 @@
+package shivani.Assignment16;
+
+import java.util.Scanner;
+
+/*Find the frequency of character from a given String array.
+Note: Take String and character from the user using scanner class.
+
+Input : technocredits
+char : e
+output : e -> 2*/
+public class CharLength {
+	static int count = 0;
```

Comment:
You should write this static variable with in method as, it’s useful for only that method.  If variable going to use multiple places then it makes senses to define that variable at static/instance. just trying to explain good coding practices.